assets/js/main.js

```diff
@@ -1,4 +1,59 @@
-jQuery( document ).ready( function( $ ) {
-  // Your JavaScript goes here
-  	console.log('Planet4 javascript reporting for duty!')
-});
+  $(window).scroll(function() {

```

Comment:
I wouldn't see any of these changes as being associated with the header block...it's something to do with the menu? If so it should be in a separate branch.

Reply:
Yes it is related to header menu. I will create another branch for header and footer markup implementation and remove it from header block branch [ feature/1265-theme-header ]